ui/src/app/login/login.tsx

```diff
@@ -24,9 +25,11 @@ function getRedirect(): string {
 }
 
 export function Login() {
+    const {t} = useTranslation();
+
     useCollectEvent('openedLogin');
     return (
-        <Page title='Login' toolbar={{breadcrumbs: [{title: 'Login'}]}}>
+        <Page title={t('login')} toolbar={{breadcrumbs: [{title: 'Login'}]}}>
```

Comment:
```suggestion
        <Page title={t('login')} toolbar={{breadcrumbs: [{title: t('Login')}]}}>
```

This might not matter for the login page, but in some cases the breadcrumbs are visible text